packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -209,40 +192,32 @@ function getTextualContent(content) {
  * @param onSelect optional, on header click callback function
  * @param onToggle optional, on caret click callback function
  * @param expanded optional, defines if the panel should be expanded or not
- * @param selected optional, defines if the panel is selected or not
  * @param theme optional, defines the theme of the collapsible, there is a default theme
+ * @param status optional, defines the status of the panel
  * @param content optional, defines the content of the panel's body:
  * if content is an array a key value content list is rendered otherwise it is a textual content
  *
  * @example
  * <CollapsiblePanel {...props} />
  */
-function CollapsiblePanel({ header, content, onSelect, onToggle, selected, expanded, theme }) {
+function CollapsiblePanel({ header, content, onSelect, onToggle, status, expanded, theme }) {
 	const headerItems = renderHeader(header, content, onSelect, onToggle);
-	const className = classNames(
-		'panel panel-default',
-		css['tc-collapsible-panel'],
-		{
-			[css['default-panel']]: !theme,
-			[css[theme]]: !!theme,
-			[css.selected]: selected,
-			selected,
-			[css.open]: expanded,
-		});
+	const className = classNames('panel panel-default', css['tc-collapsible-panel'], {
+		[css['default-panel']]: !theme,
+		[css[theme]]: !!theme,
+		[css.open]: expanded,
+		[css[getbsStyleFromStatus(status) || status]]: !!status,
+		status,
+	});
```

Comment:
Lots of complexity for some classes. I can't from a first read be sure of what makes it as class.

Reply:
@talend-glorieux I agree with you but I do not know how to improve it and I do not know why there were so much class.